docs/document/content/user-manual/shardingsphere-scaling/build.cn.md

```diff
@@ -18,7 +18,61 @@ mvn clean install -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true -Drat.skip=tr
 
 或者通过[下载页面]( https://shardingsphere.apache.org/document/current/cn/downloads/ )获取安装包。
 
-2. 解压缩 proxy 发布包，修改配置文件 `conf/server.yaml`，这里主要是开启 `scaling` 和 `mode` 配置：
+> Scaling还是实验性质的功能，在持续进化中，建议使用master分支最新版本，点击此处[下载最新版本]( http://117.48.121.24:8080/Proxy/ )
+
+2. 解压缩 proxy发布包，修改配置文件`conf/sharding.yaml`，配置`schemaName`，`dataSources`，`rules`里的`shardingAlgorithms`和`keyGenerators`，以下示例可供参考：
```

Comment:
Does `conf/sharding.yaml` mean `conf/config-sharding.yaml`?

Reply:
yes, my fault, I will change this.